metabase/metabase
# Add a way to show xrays for models in collection views

Adds a menu option to x-ray a model from collection views.

How to test:
- Open a collection with a model
- Click on the ellipsis icon -> X-ray this 
- A dashboard should appear

Screenshots:
<img width="1345" alt="Screenshot 2023-03-16 at 14 42 50" src="https://user-images.githubusercontent.com/8542534/225620492-6fd4836b-74bb-4775-a59a-effba198b2da.png">
<img width="970" alt="Screenshot 2023-03-16 at 14 42 54" src="https://user-images.githubusercontent.com/8542534/225620527-67a651f2-d54d-4b9e-b720-c1d28d8bee48.png">
